dm/master/server.go

```diff
@@ -246,14 +246,9 @@ func (s *Server) StartTask(ctx context.Context, req *pb.StartTaskRequest) (*pb.S
 				return
 			}
 			workerResp, err := cli.StartSubTask(ctx, &pb.StartSubTaskRequest{Task: stCfgToml})
-			if err != nil {
-				workerResp = &pb.CommonWorkerResponse{
-					Result: false,
-					Msg:    errors.ErrorStack(err),
-				}
-			}
-			workerResp.Worker = worker
-			workerRespCh <- workerResp
+			workerResp = s.handleOperationResult(ctx, cli, stCfg.Name, err, workerResp)
```

Comment:
some fields in `workerResp` (like `LogID`) will be overwritten by this `handleOperationResult`.
if `handleOperationResult` success it's fine, but if `handleOperationResult` failed, some fields lost.

Reply:
it wouldn't fail, definition is it that like you said - `handleOperationResult` would handle and return `response`

Reply:
```
» resume-task test-2
{
    "op": "Resume",
    "result": true,
    "msg": "",
    "workers": [
        {
            "meta": {
                "result": false,
                "worker": "127.0.0.1:8262",
                "msg": "current stage is not paused not valid\ngithub.com/pingcap/dm/dm/master.(*Server).waitOperationOk\n\t/Users/zhangxc/gopath/src/github.com/pingcap/dm/dm/master/server.go:1761\ngithub.com/pingcap/dm/dm/master.(*Server).handleOperationResult\n\t/Users/zhangxc/gopath/src/github.com/pingcap/dm/dm/master/server.go:1786\ngithub.com/pingcap/dm/dm/master.(*Server).OperateTask.func1\n\t/Users/zhangxc/gopath/src/github.com/pingcap/dm/dm/master/server.go:327\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1337"
            },
            "op": "Resume",
            "logID": "0"
        }
    ]
}
```

for the example above, `logID` lost.

Reply:
yep, you're right. I mistakenly think it is CommonWorkerResponse